app/models/osf-model.ts

```diff
@@ -226,6 +226,14 @@ export default class OsfModel extends Model {
         return this.modifyM2MRelationship('post', relationshipName, relatedModel);
     }
 
+    async removeM2MRelationship<T extends OsfModel>(
+        this: T,
+        relationshipName: RelationshipsFor<T> & string,
+        relatedModel: OsfModel | [],
+    ) {
+        return this.modifyM2MRelationship('patch', relationshipName, relatedModel);
```

Comment:
Under what circumstance would you use `removeM2MRelationship` and have anything but `[]` for `relatedModel`? 
```suggestion
        return this.modifyM2MRelationship('patch', relationshipName, []);
```